src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts

```diff
@@ -539,9 +561,7 @@ export const useSaveAndPublishSetup = (): SaveAndPublishSetup => {
     }
 
     if (createApplet.fulfilled.match(result)) {
-      Mixpanel.track('Applet Created Successfully', {
-        'Applet ID': appletId,
-      });
+      trackSave('Applet Created Successfully');
```

Comment:
I think we should include the `createdAppletId` in this call

```typescript
trackSave('Applet Created Successfully', {
  [MixpanelProps.AppletId]: createdAppletId,
});
```

The event currently shows up with the value `new-applet`.

The `Password added successfully` event also has this issue, but fixing that may be outside the scope of this PR

Reply:
Fixed both instances.